recipes/slack/webview.js

```diff
@@ -38,12 +38,23 @@ module.exports = Ferdium => {
   const getTeamIcon = function getTeamIcon(count = 0) {
     let countTeamIconCheck = count;
     let bgUrl = null;
-    const teamMenu = document.querySelector(
+    // INFO: A new Slack UI was introduced in August 2023 and will be rolled out gradually,
+    // therefore we need to support both old and new UI for the time being
+    // See more: https://slack.com/blog/productivity/a-redesigned-slack-built-for-focus
+    const oldSlackUiTeamMenu = document.querySelector(
       '#team-menu-trigger, .p-ia__sidebar_header__team_name',
     );
+    const newSlackUiTeamMenu = document.querySelector(
+      '.p-ia4_home_header_menu__button',
+    );
+
+    if (oldSlackUiTeamMenu || newSlackUiTeamMenu) {
+      if (oldSlackUiTeamMenu) {
+        oldSlackUiTeamMenu.click();
+      } else if (newSlackUiTeamMenu) {
+        newSlackUiTeamMenu.click();
+      }
```

Comment:
Haven't tested it (I don't use slack). But instead of this, can't we use an array of strings on which we pass all the query selectors (old and new)? Given that the logic for both new and old just vary on the query selector?

Reply:
Not sure how that would look like, feel free to push a commit simplifying this!

I wasn't planning on keeping this code around for the long run and remove the old slack UI code in a couple of months.

Reply:
Lets go with your fix and then change it in a couple of months (when the new UI sticks) 😄 Thank you!